.github/workflows/ci.yml

```diff
@@ -30,7 +30,7 @@ jobs:
 
       - name: Build and Test
         run: |
-          ./gradlew build
+          ./gradlew build -x bwcTestSuite
```

Comment:
Since we have a different workflow [bwcTest](https://github.com/opensearch-project/job-scheduler/blob/main/.github/workflows/bwc-test-workflow.yml) already. Excluding it here.